### ISSUE TITLE
Spectroscopy notebook: implement query for Spherex

### ISSUE THREAD

**Issue by jkrick** [nasa-fornax/fornax-demo-notebooks#348]

**Comment by afaisst** [comment]
that is correct, deep field takes very long. The alternative could be to just run aperture photometry from the image MEF fits. This is what we will do for the 'Source Discovery' IRSA tool.
